resolwe_bio/tools/tximport_summarize.R

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env Rscript
+library(argparse)
+library(GenomicFeatures)
+library(readr)
```

Comment:
Can't find where functions from this package are used.

Reply:
True, will remove.